### PR TITLE
Output log format as json in log file

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/network/IOFogNetworkInterface.java [hunk 1]
```diff
@@ -172,4 +172,14 @@ private static void setDockerBridgeInterfaceName() throws Exception {
 
         return null;
     }
+    public static String getHostName() {
+        String hostname = "";
+        try {
+            InetAddress ip = InetAddress.getLocalHost();
+            hostname = ip.getHostName();
+        } catch (UnknownHostException e) {
+            LoggingService.logWarning(MODULE_NAME, "Unable to Get hostname : " +  e.getMessage());
```
Comment (neilotoole) [inline iofog-agent-daemon/src/main/java/org/eclipse/iofog/network/IOFogNetworkInterface.java]: `Unable to Get hostname` ... should be `Unable to get hostname`

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/utils/logging/LogFormatter.java [hunk 2]
```diff
@@ -28,12 +37,27 @@
...
-		builder.append(formatMessage(record)).append('\n');
-		return builder.toString();
+		JsonBuilderFactory factory = Json.createBuilderFactory(null);
+		JsonObjectBuilder jsonObjectBuilder = factory.createObjectBuilder();
+		jsonObjectBuilder.add("timestamp", df.format(System.currentTimeMillis()));
+		jsonObjectBuilder.add("level", record.getLevel().toString());
+		jsonObjectBuilder.add("agent_id", Configuration.getIofogUuid());
+		jsonObjectBuilder.add("pid", fogNetworkInterfaceManager.getPid() != 0 ? fogNetworkInterfaceManager.getPid() : fogNetworkInterfaceManager.getFogPid());
+		jsonObjectBuilder.add("hostname", fogNetworkInterfaceManager.getHostName() != null ? fogNetworkInterfaceManager.getHostName(): IOFogNetworkInterface.getHostName());
+		String[] data = formatMessage(record).split(",");
```
Comment (baghbidi) [inline iofog-agent-daemon/src/main/java/org/eclipse/iofog/utils/logging/LogFormatter.java]: I am not sure if this is the right way do extract message/stacktrace/... 
What if any of those contain `,`?

Looking at `LogRecord` class, seems like it already has `threadID`, `thrown`, `loggerName`, `millis`, ... that we could probably use

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/network/IOFogNetworkInterface.java (hunk 1 above)
Comment (NehaNaithani) [reply iofog-agent-daemon/src/main/java/org/eclipse/iofog/network/IOFogNetworkInterface.java]: Fixed

File: iofog-agent-daemon/src/main/java/org/eclipse/iofog/utils/logging/LogFormatter.java (hunk 2 above)
Comment (NehaNaithani) [reply iofog-agent-daemon/src/main/java/org/eclipse/iofog/utils/logging/LogFormatter.java]: Updated with above comments

**Review by baghbidi — APPROVED** [review]

**Pull request merged**